### PR TITLE
TL translations for rebasing.asc

### PR DESCRIPTION
**Pull request by josteinfrancois**
Hello everyone. This is my completed translations for rebasing.asc. Feel free to review my contribution. Did 1,127 words of translations for this one.

### CODE REVIEW

File: book/03-git-branching/sections/rebasing.asc [hunk 1]
```diff
@@ -126,95 +126,73 @@ Ahh, ngunit ang kaligayahan ng pag-rebase ay mayroon ding kakulangan, na maaarin
...
+Tumingin tayo sa isang halimbawa kung papaano ang pag-rebase ng trabaho na iyong ginawang publiko ay maaaring magsanhi ng mga problema. Ipagpalagay na ikaw ay nag-clone mula sa isang sentral na server at pagkatapos ay gumawa ng ilang trabaho sa iyon. Ang iyong kasaysayan ng commit ay magmumukhang katulad nito:
 
-.Clone a repository, and base some work on it
-image::images/perils-of-rebasing-1.png["Clone a repository, and base some work on it."]
+.Mag-clone ng isang repositoryo, at magbase ng ilang trabaho nito
+image::images/perils-of-rebasing-1.png["Mag-clone ng isang repositoryo, at magbase ng ilang trabaho nito."]
 
-Now, someone else does more work that includes a merge, and pushes that work to the central server.
-You fetch it and merge the new remote branch into your work, making your history look something like this:
+Ngayon, may iba pang gumagawa ng higit pang trabaho na nagsasama ng isang merge, at nagpu-push ng trabahong iyon sa sentral na server. Nag-fetch ka nito at nag-merge ng bagong remote na branch sa iyong trabaho, ginagawang magkamukha nito ang iyong kasaysayan:
```
Comment (mind-blast) [inline book/03-git-branching/sections/rebasing.asc]: Nag-fetch ka nito  to Kinuha mo ito

File: book/03-git-branching/sections/rebasing.asc [hunk 2]
```diff
@@ -126,95 +126,73 @@ Ahh, ngunit ang kaligayahan ng pag-rebase ay mayroon ding kakulangan, na maaarin
...
 
-.Fetch more commits, and merge them into your work
-image::images/perils-of-rebasing-2.png["Fetch more commits, and merge them into your work."]
+.Mag-fetch pa ng higit pang mga commit, at i-merge ang mga ito sa iyong trabaho
```
Comment (mind-blast) [inline book/03-git-branching/sections/rebasing.asc]: Mag-fetch pa ng higit pang mga commit to Kumuha ng maraming mga commit

File: book/03-git-branching/sections/rebasing.asc [hunk 3]
```diff
@@ -126,95 +126,73 @@ Ahh, ngunit ang kaligayahan ng pag-rebase ay mayroon ding kakulangan, na maaarin
...
+image::images/perils-of-rebasing-2.png["Mag-fetch pa ng higit pang mga commit, at i-merge ang mga ito sa iyong trabaho."]
 
-Next, the person who pushed the merged work decides to go back and rebase their work instead; they do a `git push --force` to overwrite the history on the server.
-You then fetch from that server, bringing down the new commits.
+Susunod, ang tao na nag-push ng na-merge na trabaho ay nagpasyang bumalik at sa halip ay mag-rebase ng kanilang trabaho; gumawa sila ng isang `git push --force` upang i-overwrite ang kasaysayan sa server. Ikaw ngayon ay nag-fetch mula sa server na iyon, hinihila pababa ang mga bagong commit.
```
Comment (mind-blast) [inline book/03-git-branching/sections/rebasing.asc]: i-overwrite to sapawan

File: book/03-git-branching/sections/rebasing.asc [hunk 4]
```diff
@@ -126,95 +126,73 @@ Ahh, ngunit ang kaligayahan ng pag-rebase ay mayroon ding kakulangan, na maaarin
...
 
 [[_pre_merge_rebase_work]]
-.Someone pushes rebased commits, abandoning commits you've based your work on
-image::images/perils-of-rebasing-3.png["Someone pushes rebased commits, abandoning commits you've based your work on."]
+.May nagpu-push ng na-rebase na mga commit, iniiwanan ang mga commit kung saan mo binase ang iyong trabaho
```
Comment (mind-blast) [inline book/03-git-branching/sections/rebasing.asc]: May to May isang tao na

File: book/03-git-branching/sections/rebasing.asc [hunk 5]
```diff
@@ -126,95 +126,73 @@ Ahh, ngunit ang kaligayahan ng pag-rebase ay mayroon ding kakulangan, na maaarin
...
+Ito ay gumagana lamang kung ang C4 at C4' na ginawa ng iyong kasosyo ay halos eksaktong magkatugma na patch. Kung hindi ay ang rebase ay hindi makakapagsabi na ito ay isang kopya at magdaragdag ng ibang katulad ng C4 na patch (na marahil ay mabibigong malinis na maglapat, dahil ang mga pagbabago ay marahil nandoon na).
 
-You can also simplify this by running a `git pull --rebase` instead of a normal `git pull`.
-Or you could do it manually with a `git fetch` followed by a `git rebase teamone/master` in this case.
+Maaari mo ring pasimplihin ito sa pamamagitan ng isang `git pull --rebase` sa halip na isang normal na `git pull`. O maaari mong manu-manong gawin ito gamit ang isang `git fetch` na sinusundan ng isang `git rebase teamone/master` sa kasong ito.
 
-If you are using `git pull` and want to make `--rebase` the default, you can set the `pull.rebase` config value with something like `git config --global pull.rebase true`.
+Kung ikaw ay gumagamit ng `git pull` at gustong i-`--rebase` ang default, maaari mong itakda ang value ng `pull.rebase` na config gamit ang kagaya ng `git config --global pull.rebase true`.
```
Comment (mind-blast) [inline book/03-git-branching/sections/rebasing.asc]: value to halaga

File: book/03-git-branching/sections/rebasing.asc (hunk 1 above)
Comment (josteinfrancois) [reply book/03-git-branching/sections/rebasing.asc]: Will update this one. Thank you.

File: book/03-git-branching/sections/rebasing.asc (hunk 2 above)
Comment (josteinfrancois) [reply book/03-git-branching/sections/rebasing.asc]: Changing it now.

File: book/03-git-branching/sections/rebasing.asc (hunk 3 above)
Comment (josteinfrancois) [reply book/03-git-branching/sections/rebasing.asc]: noted, updating

File: book/03-git-branching/sections/rebasing.asc (hunk 4 above)
Comment (josteinfrancois) [reply book/03-git-branching/sections/rebasing.asc]: changing it now

File: book/03-git-branching/sections/rebasing.asc (hunk 5 above)
Comment (josteinfrancois) [reply book/03-git-branching/sections/rebasing.asc]: noted

**Comment by josteinfrancois** [comment]
Thank you @robin-ho for reviewing my contribution. It has been updated based on your suggestions.

File: book/03-git-branching/sections/rebasing.asc [hunk 6]
```diff
@@ -126,95 +126,73 @@ Ahh, ngunit ang kaligayahan ng pag-rebase ay mayroon ding kakulangan, na maaarin
 
 Kung susundin mo ang patnubay na iyon, magiging maayos ka. Kung hindi, kasusuklaman ka ng mga tao, at kakamuhian ka ng iyong mga kaibigan at pamilya.
 
-When you rebase stuff, you're abandoning existing commits and creating new ones that are similar but different.
-If you push commits somewhere and others pull them down and base work on them, and then you rewrite those commits with `git rebase` and push them up again, your collaborators will have to re-merge their work and things will get messy when you try to pull their work back into yours.
+Kapag ikaw ay nag-rebase ng mga bagay, ikaw ay lumilisan sa umiiral na mga commit at gumagawa ng mga panibago na kahawig ngunit naiiba. Kung magpu-push ka ng mga commit saanman at ang iba ay magpu-pull down ng mga ito at magbabase sa trabaho nito, at pagkatapos ay isusulat mo muli ang mga commit na iyon gamit ang `git rebase` at ipu-push up ang mga ito ulit, ang iyong mga katulong ay kailangang mag merge muli ng kanilang trabaho at ang mga bagay ay magugulo kapag sinubukan mong mag-pull sa kanilang trabaho patungo pabalik sa iyo.
```
Comment (heywens) [inline book/03-git-branching/sections/rebasing.asc]: * Insert line break before ``Kung magpu-push ka ng mga commit saanman``
* ``ulit`` to ``uli`` or ``muli`` -> `i-push muli ang mga ito`?

File: book/03-git-branching/sections/rebasing.asc [hunk 7]
```diff
@@ -126,95 +126,73 @@ Ahh, ngunit ang kaligayahan ng pag-rebase ay mayroon ding kakulangan, na maaarin
...
 
-Let's look at an example of how rebasing work that you've made public can cause problems.
-Suppose you clone from a central server and then do some work off that.
-Your commit history looks like this:
+Tumingin tayo sa isang halimbawa kung papaano ang pag-rebase ng trabaho na iyong ginawang publiko ay maaaring magsanhi ng mga problema. Ipagpalagay na ikaw ay nag-clone mula sa isang sentral na server at pagkatapos ay gumawa ng ilang trabaho sa iyon. Ang iyong kasaysayan ng commit ay magmumukhang katulad nito:
```
Comment (heywens) [inline book/03-git-branching/sections/rebasing.asc]: Insert line breaks before
* ``Ipagpalagay``
* ``Ang iyong kasaysayan``

File: book/03-git-branching/sections/rebasing.asc [hunk 8]
```diff
@@ -126,95 +126,73 @@ Ahh, ngunit ang kaligayahan ng pag-rebase ay mayroon ding kakulangan, na maaarin
...
+image::images/perils-of-rebasing-4.png[Ikaw ay muling nag-merge sa parehong trabaho patungo sa isang panibagong merge na commmit.]
 
-If you run a `git log` when your history looks like this, you'll see two commits that have the same author, date, and message, which will be confusing.
-Furthermore, if you push this history back up to the server, you'll reintroduce all those rebased commits to the central server, which can further confuse people.
-It's pretty safe to assume that the other developer doesn't want `C4` and `C6` to be in the history; that's why they rebased in the first place.
+Kung magpapatakbo ka ng isang `git log` kapag ang iyong kasaysayan ay magmumukhang katulad nito, makikita mo ang dalawang mga commit na mayroong parehong may-akda, petsa, at mensahe, na kung saan ay nakalilito. At saka, kung ipu-push mo ang kasaysayang ito pabalik sa server, ipapakilala mo ulit ang lahat ng mga na-rebase na commit na iyon sa sentral na server, na kung saan ay tuluyan pang nakakalito sa mga tao. Medyo ligtas magpalagay na ang ibang developer ay hindi gusto ang `C4` at `C6` sa kasaysayan; iyan ang dahilan kung bakit sila nag-rebase sa simula pa lang.
```
Comment (heywens) [inline book/03-git-branching/sections/rebasing.asc]: Put line breaks before:
* ``At saka, kung ipu-push mo``
* ``Medyo ligtas magpalagay na ang ibang developer``

File: book/03-git-branching/sections/rebasing.asc [hunk 9]
```diff
@@ -126,95 +126,73 @@ Ahh, ngunit ang kaligayahan ng pag-rebase ay mayroon ding kakulangan, na maaarin
...
+Ngayong nakita mo na ang pag-rebase at pag-merge na kumikilos, baka ikaw ay nagtataka kung ano ang mas mabuti. Bago natin maaaring sagutin ito, umatras muna tayo at pag-usapan ang tungkol sa ibig sabihin ng kasaysayan.
 
-One point of view on this is that your repository's commit history is a *record of what actually happened.*
-It's a historical document, valuable in its own right, and shouldn't be tampered with.
-From this angle, changing the commit history is almost blasphemous; you're _lying_ about what actually transpired.
-So what if there was a messy series of merge commits?
-That's how it happened, and the repository should preserve that for posterity.
+Isang pananaw nito ay ang iyong kasaysayan ng commit sa repositoryo ay isang *rekord kung ano ang tunay na nangyari.* Ito ay isang makasaysayang dokumento, mahalaga sa sarili nitong karapatan, at hindi dapat mabago. Mula sa anggulong ito, ang pagbabago sa kasaysayan ng commit ay halos lapastangan sa diyos; ikaw ay _nagsisinungaling_ tungkol sa kung ano ang tunay na naganap. Kaya ano kung mayroong isang magulong serye ng merge na mga commit? Iyon ang nangyari, at ang repositoryo ay kailangang i-preserba iyon para sa angkan.
```
Comment (heywens) [inline book/03-git-branching/sections/rebasing.asc]: line breaks:
> Ito ay isang makasaysayang
> Mula sa anggulong ito
> Kaya ano kung mayroong isang magulong serye

**Review by heywens — COMMENTED** [review]
minor suggestions

File: book/03-git-branching/sections/rebasing.asc (hunk 6 above)
Comment (josteinfrancois) [reply book/03-git-branching/sections/rebasing.asc]: changing it now

File: book/03-git-branching/sections/rebasing.asc (hunk 7 above)
Comment (josteinfrancois) [reply book/03-git-branching/sections/rebasing.asc]: adding line breaks...

File: book/03-git-branching/sections/rebasing.asc (hunk 8 above)
Comment (josteinfrancois) [reply book/03-git-branching/sections/rebasing.asc]: adding line breaks...

**Comment by josteinfrancois** [comment]
Thank you for your reviewing my work @heywens. It has been updated.

**Pull request merged**